retrieval/discovery/dns.go

```diff
@@ -110,12 +113,6 @@ func (dd *DNSDiscovery) Run(ch chan<- *config.TargetGroup) {
 
 // Stop implements the TargetProvider interface.
 func (dd *DNSDiscovery) Stop() {
```

Comment:
This method can be removed now, as it's empty and no longer part of the `TargetProvider` interface.